src/MongoMemoryReplSet.ts

```diff
@@ -219,6 +230,8 @@ export default class MongoMemoryReplSet extends events.EventEmitter {
       };
       await admin.command({ replSetInitiate: rsConfig });
       this.debug('Waiting for replica set to have a PRIMARY member.');
+      // @ts-ignore : why do we passe an Admin instance instead of an Db instance, shoud we have a more specific function
+      // for an Admin instance ?
       await this._waitForPrimary(admin);
```

Comment:
@nodkz The question in the comments :)

Reply:
@jloveridge can help with this?

Reply:
As fast solution I created `admin` property in class and rewrite a little bit `this._waitForPrimary()` method

Reply:
Ok sounds good for me !